.circleci/config.yml

```
@@ -317,7 +317,7 @@ jobs:
                 ./Tests/scripts/run_tests.sh "Demisto GA"
 
               else
-                ./Tests/lastest_server_build_scripts/run_tests.sh
+                ./Tests/scripts/run_tests.sh "Server Master"
```

taken from role of CustomBuild: https://github.com/demisto/content-test-conf/blob/master/awsinstancetool/instancesconfig.json#L475
```suggestion
                ./Tests/scripts/run_tests.sh "master"
```